spec/page_spec.rb

```diff
@@ -235,4 +235,55 @@
       wait_for { message_b }.to eq("goodbye")
     end
   end
+
+  describe "#resize" do
+    def body_size
+      {
+        height: page.evaluate("document.body.clientHeight"),
+        width: page.evaluate("document.body.clientWidth")
+      }
+    end
+
+    def is_mobile?
```

Comment:
This doesn't _really_ test whether mobile emulation is enabled, but I couldn't find a good way of doing that directly via JavaScript evaluation.  So we use touch emulation as a proxy.